mod/src/main/java/kr/syeyoung/dungeonsguide/mod/dungeon/roomfinder/DungeonRoomInfoRegistry.java

```diff
@@ -37,7 +37,21 @@
 import java.security.NoSuchAlgorithmException;
 import java.util.*;
 
+
+
 public class DungeonRoomInfoRegistry {
+
+    /**
+     * DEV_USERS is used to check if the user is a dev user.
+     * You do not need to be in this list to use the commands if you are in a dev env.
+     * This lists whitelists users to allow them to use commands when not in a dev env.
+     */
+    private static final List<String> DEV_USERS =
+            Arrays.asList(
+                    "e686fe0aab804a71ac7011dc8c2b534c", //syeyoung
```

Comment:
Ehhhhhhh, the limit was intentional to prevent extraction of supporter roomdatas to normal roomdatas

but i guess it will work